template/assets/use_case/template/submission/q3as_demo.ipynb

```diff
@@ -0,0 +1,132 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Getting an API Key\n",
+    "\n",
+    "To run your algorithms in the cloud, you have to create an API key and load it into your Credentials\n",
+    "\n",
+    "Start by visiting https://q3as.aqora.io and signing in with your GitHub or Google account. Click on your profile in the top right and go to API Keys. Tap Add API Key and enter a description for your API key. Tap Copy id and secret and paste them in the notebook. \n",
+    "\n",
+    "\n",
+    "> ⚠️ **Warning: Be Cautious with Secrets!**\n",
```

Comment:
Maybe remove this? It seems unnecessary and scary

Reply:
Since you recommend using a .env file, I wouldn't remove it. I mean, sometimes users might accidentally push credentials!